exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillFunctionRegistry.java

```diff
@@ -122,6 +127,13 @@ public int size(){
   }
 
   public void register(DrillOperatorTable operatorTable) {
+    registerForInference(operatorTable);
+    registerForDefault(operatorTable);
```

Comment:
Renamed one as registerOperatorsWithoutInference; and the other as registerOperatorsWithInference